components/FinancialSummary/ContactUs.tsx

```diff
@@ -0,0 +1,36 @@
+import { HeadingLevel, HeadingTypeStyle } from '@/types/typography/Heading';
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import Button from '../buttons/Button';
+import Heading from '../typography/Heading';
+import Paragraph from '../typography/Paragraph';
+import TextLink from '../typography/TextLink';
+
+/**
+ * ContactUs component displays information about getting in touch.
+ * {JSX.Element} ContactUs component.
```

Comment:
Kindly add `@description` and remove the return `JSX.Element`. Check the suggestions above.